lib/spack/spack/util/environment.py

```diff
@@ -40,11 +40,13 @@ def env_flag(name):
     return False
 
 
+# FIXME : remove this function ?
```

Comment:
I think it makes sense to keep these, as there are still good reasons a package developer might want to use them in `install()`.  Mainly, if they want to make environment changes *only* for the build that *won't* go into a module. Modifications to `CC`, `CXX`, etc. probably fall into this category most of the time.